guides/posts/pratham/create-a-text-sentiment-analysis-app/post.md

```diff
@@ -0,0 +1,318 @@
+---
+title: "How to Create a Text Sentiment Analysis App using React"
+description: "Text sentiment analysis is a machine learning and natural language processing technique to analyze the text sentiment of a writer. The sentiment can be positive, negative, or neutral. Text sentiment analysis, also known as opinion mining, is a deep area of research. In this blog, we will make a text sentiment analysis application using an API available on RapidAPI Hub."
+slug: "2021-09-23-post-5"
+authors: ["pratham"]
+category: "API"
+tags: ["react", "api"]
+publishedDate: "2021-09-23T08:00:00+08:00"
+coverImage: ""
+---
+
+<Lead>
+  Text sentiment analysis is a machine learning and natural language processing technique to analyze the text sentiment of a writer. The sentiment can be positive, negative, or neutral. Text sentiment analysis, also known as opinion mining, is a deep area of research. In this blog, we will make a text sentiment analysis application using an API available on RapidAPI Hub.
+</Lead>
+
+[Text Analysis](https://rapidapi.com/gaurmanojkumar530/api/text-analysis12/?utm_source=medium.com%2F%40Rapid_API&utm_medium=DevRel&utm_campaign=DevRel) is an excellent API available on [RapidAPI Hub](https://rapidapi.com/hub?utm_source=medium.com%2F%40Rapid_API&utm_medium=DevRel&utm_campaign=DevRel) using which you can create a text analysis application. [Text Analysis](https://rapidapi.com/gaurmanojkumar530/api/text-analysis12/?utm_source=medium.com%2F%40Rapid_API&utm_medium=DevRel&utm_campaign=DevRel) has several other endpoints as well which let you perform sentiment-analysis, text-summarization, language-detection, article-extraction, named-entity-recognition on text. You can also analyze text from a whole website or from documents and images using this API.   
+
+We will be focusing on the text analysis endpoint in this particular article. Let’s get started. 
+
+## How to connect to the Facial Text Analysis API
+
+RapidAPI handles all the API development things very effectively. You don’t need to worry about any complicated stuff related to API like rate-limiting factors, authentication, authorization. RapidAPI does all the communications by itself. You just need to register on RapidAPI Hub if you want to use any API. RapidAPI provides you a single API key using which you can access over 35,000 APIs conveniently.   
+
+Go to [RapidAPI Hub](https://rapidapi.com/hub?utm_source=medium.com%2F%40Rapid_API&utm_medium=DevRel&utm_campaign=DevRel) and create a free account.
+
+<figure>
```

Comment:
Use [the markdown image syntax](https://daringfireball.net/projects/markdown/syntax#img).